src/styles/admin.scss

```diff
@@ -0,0 +1,39 @@
+body {
+   font-family: "Roboto Condensed", sans-serif;;
+  }
+ 
+.pill{ 
+   border-radius: 0;
+}
+
+.btn--style-secondary {
+  border-radius: 0;
+}
+
+.card__click{ 
+   border: 1px solid #c60c1a;
+   border-radius: 0;
+}
+
+.icon--plus {
+   color: #c60c1a;
+}
+
+.btn--style-primary {
+   color: #c60c1a;
+   background-color: transparent;
+   border: 2px solid #c60c1a;
+   border-radius: 0;
+}
+
+.btn + .btn--style-primary:hover {
+   color: #ffffff;
```

Comment:
Keine Ahnung, ob man das so macht, aber du kannst in so einem Fall abkürzen: `#fff` 
https://developer.mozilla.org/en-US/docs/Web/CSS/hex-color
z.B. "#ABC"  steht für "#AABBCC"